### PR TITLE
gurbani text file work and default path change

### PR DESCRIPTION
**Pull request by irvanjit**

### CODE REVIEW

File: www/src/scss/_theme-light.scss [hunk 1]
```diff
@@ -36,8 +36,7 @@ $alt-bg: #fafafa;
 $dialog-bg: #fff;
 $base-text: #4e4e4e;
 
-.light-theme,
-.custom-theme {
```
Comment (tsingh777) [inline www/src/scss/_theme-light.scss]: bad merge? pretty sure this is needed.

File: www/src/scss/_theme-moody-blue.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 body.moody-blue {
   background-color: #38987b;
+  background-image: url("../../../assets/custom_backgrounds/blue.png");
```
Comment (tsingh777) [inline www/src/scss/_theme-moody-blue.scss]: duplicate line

File: app.js [hunk 3]
```diff
@@ -243,6 +244,20 @@ ipcMain.on('clear-apv', () => {
   }
 });
 
+function createBroadcastFiles(arg) {
+  const userDataPath = (electron.app || electron.remote.app).getPath('userData');
```
Comment (navdeepsinghkhalsa) [inline app.js]: `const userDataPath = electron.app.getPath('userData');`

File: www/src/scss/_theme-light.scss (hunk 1 above)
Comment (navdeepsinghkhalsa) [reply www/src/scss/_theme-light.scss]: @tsingh777 This is fine, though you're right it looks wonky. It's just deleting `.custom-theme`

**Comment by navdeepsinghkhalsa** [comment]
package-lock.json still is reporting changes, but I will regenerate it due to lack of time

**Pull request merged**